versioned_docs/version-8.3/self-managed/zeebe-deployment/operations/resource-planning.md

```diff
@@ -148,3 +156,24 @@ You should assign idempotent ids to events in your exporter if this is an issue
 ### Effect of quorum loss
 
 If a partition goes under quorum (for example, if two nodes in a 3-node cluster go down), the leader of the partition continues to accept requests, but these requests are not replicated and are not marked as committed. In this case, they cannot be truncated. This causes the event log to grow. The amount of disk space needed to continue operating in this scenario is a function of the broker throughput and the amount of time to quorum being restored. You should ensure your nodes have sufficient disk space to handle this failure mode.
+
+## Memory
+
+Memory usage is based on the Java heap size (by default [25% of the max RAM](TODO: Insert documentation here)) and native memory usage (also by default [25% of the max RAM](TODO: Insert documentation here)), so Java itself will use **up to** 50% of the maximum RAM.
```

Comment:
This is from the JVM: https://docs.oracle.com/javase/8/docs/technotes/tools/unix/java.html

The max native memory usage is typically not documented AFAIK, but we found from the OpenJDK code that if omitted, it just defaults to the same value as the max RAM percentage.

Reply:
lol what 😂